tests/foreman/conftest.py

```diff
@@ -389,3 +396,77 @@ def module_gce_compute(module_org, module_location):
         zone=settings.gce.zone, organization=[module_org],
         location=[module_location]).create()
     return gce_cr
+
+
+@pytest.fixture(scope='module')
+def azurerm_settings():
```

Comment:
Do we really need that as global fixture? Can't you have it as fixture in `tests/foreman/api/test_computeresource_azurerm.py`, or even function there?

The same applies to other fixtures below.

Reply:
Yeah I would like these to be part of `conftest` and wherever it would need further can utilize at any test level.

Reply:
I would rather put that at the most specific level and move to more general levels if we actually have something that uses it, rather than the other way around. You know, YAGNI.

Reply:
I am fine to put them at specific level as well however the approach is used as similar to:
https://github.com/SatelliteQE/robottelo/blob/80beb97d8762146893c5c1b45f9e0aedab942539/tests/foreman/conftest.py#L364-L398

Reply:
Yes, and in that PR I also suggested to put these fixtures closer to where they are used.

Reply:
Hummm ... I have seen in another PR https://github.com/SatelliteQE/robottelo/pull/7335 @jyejare where he is putting them separately now , So if that approach looks good then I'll put same there ..
but yeah for that I'll raise another PR.